src/js/hexbin/HexbinLayer.js

```diff
@@ -302,10 +302,30 @@ L.HexbinLayer = (L.Layer ? L.Layer : L.Class).extend({
 		this.options.onclick = clickFn;
 		//this._redraw();
 		return this;
+	},
+
+	/*
+	 * Returns an array of the points in the path, or nested arrays of points in case of multi-polyline.
+	 */
+	getLatLngs: function () {
+		return this._data.map(function(d) { return L.latLng(d[0], d[1]); });
+	},
+
+	/*
+	 * Get path geometry as GeoJSON
+	 */
+	toGeoJSON: function () {
+		var latLngs = this._data.map(function(d) { return L.latLng(d[0], d[1]); });
```

Comment:
But the array of LatLngs is always derived from calls to LatLng(lat, lng), so it will never have a multidimensional array, right? On a side note, I'd be super curious to see your visualization if you can share a gif or something.